develop/api_udf.rst

```diff
@@ -129,6 +137,25 @@ This example distributes a ``github_events`` table and then undistributes it.
   -- undo that and make it local again
   SELECT undistribute_table('github_events');
 
+
+.. _remove_local_tables_from_metadata:
+
+remove_local_tables_from_metadata
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+remove_local_tables_from_metadata() function removes local tables from Citus metadata
```

Comment:
This section doesn't tell me much. I have two questions:

* Why/when would someone need to call remove_local_tables_from_metadata() manually? Doesn't Citus take care of this when the foreign keys are dropped?
* What are the disadvantages of having tables stored in the metadata? Why should a user care?

Reply:
>Doesn't Citus take care of this when the foreign keys are dropped?

Yes, Citus actually does that as long as `enable_local_reference_foreign_keys` guc is set to on (by default).
So we thought that it might to better to mention this udf in docs in case that user disabled the guc or something went wrong.
What do you think ?

Reply:
OK, so something like this might provide more motivation:

```rst
The remove_local_tables_from_metadata() function removes local tables
from Citus metadata that no longer need to be there. (See
:ref:`enable_local_ref_fkeys`.)

Usually if a local table is in Citus' metadata, there's a reason, such as
the existence of foreign keys between the table and a reference table.
However, if ``enable_local_reference_foreign_keys`` is disabled, Citus
will no longer manage metadata in that situation, and unnecessary
metadata can persist.
```